addons/board/static/src/add_to_board/add_to_board.js

```diff
@@ -44,7 +44,7 @@ export class AddToBoard extends Component {
         };
 
         const result = await this.rpc("/board/add_to_dashboard", {
-            action_id: this.env.config.actionId,
+            action_id: this.env.config.actionId || false,
```

Comment:
doesn't seem necessary then, right?

Reply:
just added it in case the component is used somewhere else in the future but in this case you're right it won't displayed anyways if there's no action Id